sqlpage/templates/card.handlebars

```diff
@@ -20,10 +21,10 @@
         row-cols-xl-5
     {{/if}}
     gx-2 gy-2
-    mt-1 mb-3">
+    mt-1 mb-3 {{class}}">
     {{#each_row}}
         <div class="col">
```

Comment:
Wouldn't you like to allow giving individual cards an id ? This would allow linking to one card in particular.